### PR TITLE
refactor: split cli string files into three and extract cli description strings into string files

### PR DESCRIPTION
**Pull request by jayzhang**
`src/resource/errors.json`: Error message strings
`src/resource/commands.json`: Command description strings
`src/resource/strings.json`: Other CLI strings

### CODE REVIEW

**Review by KennethBWSong — APPROVED** [review]

**Review by Alive-Fish — APPROVED** [review]

**Review by xzf0587 — APPROVED** [review]

**Review by nliu-ms — APPROVED** [review]

**Review by swatDong — APPROVED** [review]

File: packages/cli/src/resource/strings.json [hunk 1]
```diff
@@ -1,34 +1,16 @@
 {
-  "account.login.azure": "You have successfully logged into Azure.",
-  "account.login.m365": "You have successfully logged into Microsoft 365.",
+  "account.login.azure": "Successfully signed into Azure.",
+  "account.login.azure.fail": "Failed to sign into Azure.",
```
Comment (supkasar) [inline packages/cli/src/resource/strings.json:3]: Proposed: Unable to sign into Azure

File: packages/cli/src/resource/strings.json [hunk 2]
```diff
@@ -1,34 +1,16 @@
 {
-  "account.login.azure": "You have successfully logged into Azure.",
```
Comment (supkasar) [inline packages/cli/src/resource/strings.json:2]: We use contractions - You've

File: packages/cli/src/resource/strings.json [hunk 3]
```diff
@@ -1,34 +1,16 @@
...
-  "account.login.m365": "You have successfully logged into Microsoft 365.",
```
Comment (supkasar) [inline packages/cli/src/resource/strings.json:3]: You've

File: packages/cli/src/resource/strings.json [hunk 4]
```diff
@@ -1,34 +1,16 @@
...
+  "account.login.azure": "Successfully signed into Azure.",
+  "account.login.azure.fail": "Failed to sign into Azure.",
+  "account.logout.azure": "Successfully signed out of Azure.",
+  "account.logout.azure.fail": "Failed to sign out of Azure.",
```
Comment (supkasar) [inline packages/cli/src/resource/strings.json:5]: Unable to sign out of Azure.

File: packages/cli/src/resource/strings.json [hunk 5]
```diff
@@ -1,34 +1,16 @@
...
+  "account.login.m365": "Successfully signed into Microsoft 365.",
+  "account.login.m365.fail": "Failed to sign into Microsoft 365.",
```
Comment (supkasar) [inline packages/cli/src/resource/strings.json:7]: Unable to sign into Microsoft 365.

File: packages/cli/src/resource/strings.json [hunk 6]
```diff
@@ -1,34 +1,16 @@
...
+  "account.logout.m365": "Successfully signed out of Microsoft 365.",
+  "account.logout.m365.fail": "Failed to sign out of Microsoft 365.",
```
Comment (supkasar) [inline packages/cli/src/resource/strings.json:9]: Unable to sign out of Microsoft 365.

Comment (jayzhang) [reply packages/cli/src/resource/strings.json:9]: Thanks for comments, I will create a new PR to fix this on the whole.

**Review by supkasar — APPROVED** [review]

**Pull request merged**